### DISCUSSION TITLE
Tiling mode similar to PaperWM?

### DISCUSSION THREAD

**Discussion by nobeh** [rxhanson/Rectangle#813] [Q&A] [answered]
I am setting up a Mac and actually switching from a Linux. In Linux, I've been a user of auto-tiling/scrolling window managers and esp. recently [PaperWM][1]. The main functionality is to put the next new window next to the current one in the auto-tiling mode. 

Is Rectangle (or its pro version) offering such behavior or would it be possible to configure as such?

[1]: https://github.com/paperwm/PaperWM/#scrollable-window-tiling

**Comment by rxhanson** [comment, marked as answer]
There's nothing for auto-tiling in Rectangle or Pro at this time. I have plans to add it to Pro eventually but have no committed timeframe for it. The reason that I haven't prioritized it is because there are some macOS limitations that might make the outcome less desirable than what a lot of users are used to in Linux, and it would require significant effort to see what I can workaround and how.

**Reply by nobeh** [reply]
It'd be great if you agree to have an issue to follow this feature request? Thanks

**Reply by rxhanson** [reply]
Thanks for asking before creating one. I only use issues for things that need fixing, and I convert issues that are feature requests to discussions. Having an issue won't increase the odds or decrease the timeframe for getting this implemented, but I do check back and see which discussions have more upvotes to determine which items are more desired.
